src/test/java/uk/gov/companieshouse/confirmationstatementapi/controller/FilingControllerTest.java

```diff
@@ -3,22 +3,33 @@
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.InjectMocks;
+import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
-import uk.gov.companieshouse.api.model.transaction.Transaction;
+import uk.gov.companieshouse.api.model.filinggenerator.FilingApi;
+import uk.gov.companieshouse.confirmationstatementapi.exception.SubmissionNotFoundException;
+import uk.gov.companieshouse.confirmationstatementapi.service.FilingService;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.Mockito.when;
 
 @ExtendWith(MockitoExtension.class)
 class FilingControllerTest {
 
+    private static final String CONFIRMATION_ID = "abc123";
+
     @InjectMocks
     private FilingController filingController;
+
+    @Mock
+    private FilingService filingService;
+
     @Test
-    void getFiling() {
-        var transaction = new Transaction();
-        transaction.setDescription("12345678");
-        var result = filingController.getFiling(transaction);
+    void getFiling() throws SubmissionNotFoundException {
```

Comment:
we should also test the exception paths through the controller method as well.
test for SubmissionNotFoundException and a non-SubmissionNotFoundException